### DISCUSSION TITLE
Domain nameservers "Invalid configuration"

### DISCUSSION THREAD

**Discussion by kennedyjustin** [vercel/community#1884] [Help] [answered]
### Problem Description

This has worked for me in the past, but now there seems to be an issue with vercel nameservers:
1. I own a domain through Namecheap
2. I configure nameservers in Namecheap to be `ns1.vercel-dns.com`, and `ns1.vercel-dns.com`
3. I hook up a project through Vercel, and try to configure a custom domain for it.
4. Vercel shows "Invalid configuration" for my domain:

<img width="1035" alt="Screen Shot 2023-03-25 at 10 39 17 PM" src="https://user-images.githubusercontent.com/46265478/227752348-3950e7d8-8812-420a-bb1b-9025f15ffded.png">

Do I just need to wait longer? Its been over an hour, and previously this has always been near-instantaneous.

I will note that when I configure the domain via A and CNAME records, everything works fine.

I prefer to use Vercel as my name server, but if its going to be this jank maybe i'll just stick to Namecheap

**Comment by sostenesapollo** [comment]
Same for me.

**Comment by amyegan** [comment, marked as answer]
It may take 24-48 hours for changes to fully propagate. You can check DNS propagation status using a tool like [whatsmydns.net](https://www.whatsmydns.net/) or `dig`

Our documentation and help guides have more info:
- [Why is my Vercel domain not verified?](https://vercel.com/guides/why-is-my-vercel-domain-unverified)
- [DNS propagation](https://vercel.com/docs/concepts/projects/domains#dns-propagation)
- [Troubleshooting domains](https://vercel.com/docs/concepts/projects/domains/troubleshooting#)
